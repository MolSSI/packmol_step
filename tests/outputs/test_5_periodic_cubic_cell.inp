tolerance 2.0
output packmol.pdb
filetype pdb
connect yes
structure input_1.pdb
   inside cube 0.0 0.0 0.0 19.9053
   number 6
end structure
structure input_2.pdb
   inside cube 0.0 0.0 0.0 19.9053
   number 300
end structure
