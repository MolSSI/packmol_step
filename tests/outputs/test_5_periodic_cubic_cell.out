    Will create a cubic region containing the following molecules:

        +-------------+------------------+---------+
        |  Component  | Structure        |   Ratio |
        |-------------+------------------+---------|
        |    fluid    | c1ccccc1c2ccccc2 |       1 |
        |    fluid    | O                |      50 |
        +-------------+------------------+---------+

    The dimensions of the region will be calculated from the density 1.0 g/ml.
    The number of molecules of the fluid will be obtained by rounding 1000 atoms
    to give a whole number of molecules with the requested ratios.

Created a periodic cubic cell 21.91 Ã… on a side with 306 fluid molecules

    +-------------+------------------+----------+
    |  Component  | Structure        |   Number |
    |-------------+------------------+----------|
    |    fluid    | c1ccccc1c2ccccc2 |        6 |
    |    fluid    | O                |      300 |
    +-------------+------------------+----------+

There are a total of 1032 atoms in the cell giving a density of 1.0 g/ml.